typescript/cli/src/deploy/TestRecipientDeployer.ts

```diff
@@ -0,0 +1,63 @@
+import debug from 'debug';
+
+import { TestRecipient, TestRecipient__factory } from '@hyperlane-xyz/core';
+import {
+  ChainName,
+  HyperlaneDeployer,
+  MultiProvider,
+} from '@hyperlane-xyz/sdk';
+import { Address, eqAddress } from '@hyperlane-xyz/utils';
+
+export type TestRecipientConfig = {
+  interchainSecurityModule: Address;
+};
+
+export type TestRecipientContracts = {
+  testRecipient: TestRecipient;
+};
+
+export type TestRecipientAddresses = {
+  testRecipient: Address;
+};
+
+export const testRecipientFactories = {
+  testRecipient: new TestRecipient__factory(),
+};
+
+export class TestRecipientDeployer extends HyperlaneDeployer<
```

Comment:
do we not have this int he SDK already?

Reply:
You'd think so but no. There's an older version in infra which I can hopefully kill soon. Though I agree this belongs in the SDK. Leaving a TODO which I'll address in one of my my infra migration PRs